build.py

```diff
@@ -556,7 +556,7 @@ def Package(context):
                     help="Directory where Maya is installed.")
 
 parser.add_argument("--mayausd-location", type=str,
-                    help="Directory where MayaUsd is installed.")
+                    help="Directory where mayaUSD.mod file is.")
```

Comment:
Actually I think the previous help was correct. This flag should be used to specify where the MayaUsd install is. Then you should create a `cmake/modules/FindMayaUsd.cmake` file where search for the new api library you want to link with and the include folder (setting two new cmake variables for these (similar to FindUfe.cmake), such as: MAYAUSD_INCLUDE_DIR and MAYAUSD_API_LIBRARY.

I think you could also add cmake code in this new FindMayaUsd.cmake that looks for the mayaUsd.mod file and if found adds the path to MAYA_MODULE_PATH.